### DISCUSSION TITLE
SOC2 compliance

### DISCUSSION THREAD

**Discussion by sahilpaayj** [supabase/supabase#21165] [Questions]
I saw that SOC2 compliance is included in the Supabase "Team" plan. What does that entail? Does that just mean Supabase Team can be used in a SOC2 Type 2 compliant way? Or does it mean that Supabase audit and provide a report themselves?

reference: https://supabase.com/pricing ,  https://supabase.com/blog/supabase-soc2-hipaa

**Comment by arladmin** [comment]
👆

**Comment by GaryAustin1** [comment]
I believe, I am not an employee, that Team plan gives you access to the report itself to prove they are complaint.  This would be something  your business or your customers may require.   It has no bearing on if they are compliant or not for your use of their system.  They represent they are.
Does your business require proof in the form of a document?

**Comment by sahilpaayj** [comment]
I am not sure if proof is required, but it would be good to see the SOC2 report to confirm. 
If that is the offering, I feel like that should be available to all tiers of subscription.
